server/controllers/pwd.controllers.js

```diff
@@ -0,0 +1,99 @@
+const asyncHandler = require('express-async-handler');
+const User = require('../models/user.modal');
+const nodemailer = require('nodemailer');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+
+module.exports.forgotPwdController = asyncHandler(async (req, res, next) => {
+  const { email } = req.body;
+  const user = await User.findOne({ email });
+
```

Comment:
Before generating token we should check lastupdate date&time which should be more than 10 min, if not then send 202 status code with the message "You can do reset password after 10 Min".

Reply:
Also in the past user doesn't have the last update field so take care of it.

Reply:
> Before generating token we should check lastupdate date&tim

This i have done ,could u review line72

Reply:
No, I am talking about while generating tokens. SO that the user can't spam it multiple times.

Reply:
Yeah ok so if a user clicks forgot password, he can't raise another token for 10 mins right?

Reply:
Yes

Reply:
Hey done, added rate limiter :-)